pkg/router/websocket_test.go

```diff
@@ -0,0 +1,73 @@
+package router_test
```

Comment:
Test coverage for the patch is low. Actually around 50%. Please add tests so that we will have test coverage of the entire patch.